### PR TITLE
Loading partial

### PR DESCRIPTION
**Pull request by ncounter**
The simple ` Loading <something> ...` with the spinning icon is a pattern to be reused: this PR makes it a reusable partial.

PS: this depends on https://github.com/openSUSE/open-build-service/pull/15887 , because that PR introduces two occurrences of this partial. Only the last commit is the target of this current PR

### CODE REVIEW

File: src/api/app/views/webui/shared/_loading.haml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.text-center.p-4{ class: wrapper_css ||= '' }
```
Comment (saraycp) [inline src/api/app/views/webui/shared/_loading.haml]: I'd say this is what we need:

```suggestion
.text-center.p-4{ class: wrapper_css || '' }
```

**Review by saraycp — COMMENTED** [review]
Is there a reason to use a partial and not a ViewComponent? I pushed a commit with the ViewComponent alternative and the fix I commented on inline.
Feel free to discard my commit if it doesn't convince you.

**Comment by danidoni** [comment]
> Is there a reason to use a partial and not a ViewComponent? I pushed a commit with the ViewComponent alternative and the fix I commented on inline. Feel free to discard my commit if it doesn't convince you.

I believe the logic behind using a ViewComponent should be encapsulating complexity to allow reuse and testing. In this case there is almost no logic involved. Using a view component here introduces a bit of overhead that it's not worth it.

**Comment by saraycp** [comment]
> > Is there a reason to use a partial and not a ViewComponent? I pushed a commit with the ViewComponent alternative and the fix I commented on inline. Feel free to discard my commit if it doesn't convince you.
> 
> I believe the logic behind using a ViewComponent should be encapsulating complexity to allow reuse and testing. In this case there is almost no logic involved, so there will be no testing as well. Using a view component here introduces a bit of overhead that it's not worth it.

From the [documentation](https://viewcomponent.org/#tldr): _"ViewComponents work best for templates that are reused  [...]"_  But I understand what you explain.

**Comment by saraycp** [comment]
@danidoni @ncounter Understood! I removed the ViewComponent commit and pushed a new one with the fix commented on [this line](https://github.com/openSUSE/open-build-service/pull/15895#discussion_r1546817776).

**Comment by ncounter** [comment]
> @danidoni @ncounter Understood! I removed the ViewComponent commit and pushed a new one with only the fix commented on [this line](https://github.com/openSUSE/open-build-service/pull/15895#discussion_r1546817776).

Awesome, and thank you for the fix :slightly_smiling_face:

**Review by saraycp — APPROVED** [review]

**Pull request merged**